src/.clang-format

```diff
@@ -82,8 +104,18 @@ PenaltyBreakString: 1000
 PenaltyExcessCharacter: 1000000
 PenaltyReturnTypeOnItsOwnLine: 60
 
+# includes
+IncludeCategories: # revisit
+  - Regex: '.*'
+    Priority: 1
+IncludeIsMainRegex: '$'
+SortIncludes: false # revisit
```

Comment:
I turned it off because I need to configure the priorities and see if I can make the guards work.